jbpm-services/jbpm-services-ejb/jbpm-services-ejb-impl/src/test/java/org/jbpm/services/ejb/test/RuntimeDataServiceEJBIntegrationTest.java

```diff
@@ -817,6 +817,9 @@ public void testGetTaskById() {
     	assertEquals(processInstanceId, userTask.getProcessInstanceId());
     	assertEquals("Write a Document", userTask.getName());
 
+		NodeInstanceDesc nodeInstanceDesc = runtimeDataService.getNodeInstanceForWorkItem(userTask.getWorkItemId());
+		assertEquals(nodeInstanceDesc.getSlaCompliance(), userTask.getSlaCompliance());
+		assertEquals(nodeInstanceDesc.getSlaDueDate(), userTask.getSlaDueDate());
```

Comment:
Looks like we change UserTaskInstance class but we test nodeInstance ... shouldn’t we test usertask?

Reply:
@mswiderski here we rely here in  runtimeDataService.getNodeInstanceForWorkItem(userTask.getWorkItemId()) returns the good values for sla info. This test try to assert that the getTaskById filled new sla values are equals that the ones returned at nodeInstanceDesc

Reply:
ah ok, now I see, thanks